pkg/operator/ceph/object/controller.go

```diff
@@ -47,7 +48,8 @@ import (
 )
 
 const (
-	controllerName = "ceph-object-controller"
+	controllerName    = "ceph-object-controller"
+	bucketProvisioner = "rook-ceph.ceph.rook.io-bucket"
```

Comment:
We cannot hardcode this the `rook-ceph` part is the namespace. This won't always be the case.
Just keep `.ceph.rook.io-bucket `

Reply:
Also, you need to add this https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/bucket/util.go#L47 for backward compatibility, please export it and re-use it here.